RELEASE_NOTES.md

```diff
@@ -1,5 +1,8 @@
 # TypeDB Jupyter connector
 
+## Version 0.5
```

Comment:
As version 0.4 hasn't been released to PyPI yet, these changes would come under 0.4 rather than 0.5.